app.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Wheelie Slack App",
+    "description": "Wheelie is a Slack App originally written for WeAllJS.com.",
+    "repository": "https://github.com/WeAllJS/wheelie-slack-app",
+    "keywords": ["node", "express", "slack", "bot"],
+    "env": {
+        "SEND_WELCOME_MESSAGE": {
+            "description": "If the value of is 'FALSE', no message will be sent to new members when they join "
+        },
+        "WELCOME_MESSAGE_TEXT": {
+            "description": "A message that will be send to new members once they join the group"
+        },
+        "SLACK_CLIENT_ID" : {
+            "description": ""
+        },
+        "SLACK_CLIENT_SECRET" : {
+            "description": ""
+        },
+        "VERIFICATION_TOKEN" : {
+            "description": ""
+        }
+    }
+}
```

Comment:
I don't know what exactly goes there, would you mind helping with that text, I don't wanna confuse the users.